portfolio/src/main/webapp/index.html

```diff
@@ -7,8 +7,7 @@
     <script src="script.js"></script>
 
```

Comment:
Remove extra white space lines within the same block of code.

Reply:
Thanks William, just fixed it!